### PR TITLE
Bugfix for charts with dependencies and make pre-commit hook function on pre-commit.ci

### PR DESCRIPTION
**Pull request by consideRatio**
The changes in this PR:
- fixed a bug about running `helm dep up .` too late
- removed a call to `helm repo update` that isn't relevant for `helm dep up .` as `helm dep up .` will get the helm chart repo as defined in the dependencies list no matter what.
- made the defined pre-commit hook function on pre-commit.ci by setting up a conda environment that installs `helm`

Please feel free to push commits directly to this PR or close it and use it as inspiration etc.

---

I've tested using pre-commit against a fork with these changes and it works even though I didn't have `helm` available or content in a `charts/` folder pre-populated.

## IMPORTANT

Note that this PR is a PR to the main branch, not the default branch of `branch-0.6`. See #45 for how this has confused me.

UPDATE: This wasn't the right thing to do. Closing this in favor of #46.

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

File: environment.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# This environment can provide `helm` via the kubernetes-helm package, but must
+# also explicitly install frigate itself within this repo.
+#
+name: pre-commit-hook-environment
+dependencies:
+  - python
+  - kubernetes-helm
+  - pip
+  - pip:
+    - "."
```
Comment (consideRatio) [inline environment.yml:13]:
```suggestion
      - "."
```

**Comment by jacobtomlinson** [comment]
This looks good to me! Could you retarget this to the default branch though? We only merge to `main` on release.

**Comment by consideRatio** [comment]
Thanks for the review and input about branches @jacobtomlinson! I opened #46!

**Pull request closed without merging**